core/xs_protocol.ml

```diff
@@ -37,7 +37,7 @@ module Op = struct
     | Isintroduced
     | Resume
     | Set_target
-    | Restrict
+    | Invalid
```

Comment:
It's not obvious to me why this value is introduced, nor why it's associated with a deprecation failure. Could you explain this?

Reply:
The interface defined in C has a hole where Restrict used to be, and we need to represent that (invalid) value somehow. This is how it's also treated in oxenstored.

Reply:
That makes sense, could you add the explanation in the commit message?